src/main/java/org/jabref/gui/libraryproperties/general/GeneralPropertiesViewModel.java

```diff
@@ -61,6 +66,16 @@ public void setValues() {
         selectedDatabaseModeProperty.setValue(metaData.getMode().orElse(BibDatabaseMode.BIBLATEX));
         generalFileDirectoryProperty.setValue(metaData.getDefaultFileDirectory().orElse("").trim());
         userSpecificFileDirectoryProperty.setValue(metaData.getUserFileDirectory(preferencesService.getFilePreferences().getUserAndHost()).orElse("").trim());
+        userSpecificFileDirectoryProperty.setValue(metaData.getUserFileDirectory(preferencesService.getFilePreferences().getUserAndHost()).map(path -> usernameProperty.getValue() + ": " + path).orElse("").trim());
+        String username = preferencesService.getFilePreferences().getUserAndHost();   // get the username
```

Comment:
This return username AND hostname. -- You can add new methods to the getFilePreferences to return the username and the hostname separately. - The `getUserAndHost` should IMHO be still be avialble. Check the usages.